test/adjoint.jl

```diff
@@ -36,6 +36,8 @@ easy_res4 = adjoint_sensitivities(solb,Vern9(),dg,t,abstol=1e-14,
                                   reltol=1e-14,iabstol=1e-14,ireltol=1e-12,sensealg=SensitivityAlg(backsolve=true))
 easy_res5 = adjoint_sensitivities(sol,Kvaerno5(nlsolve=NLAnderson(), smooth_est=false),dg,t,abstol=1e-14,
                                  reltol=1e-14,iabstol=1e-14,ireltol=1e-12,sensealg=SensitivityAlg(backsolve=true))
+easy_res6 = adjoint_sensitivities(solb,Vern9(),dg,t,abstol=1e-14,
```

Comment:
should stress this test more. Make it only have a checkpoint at 0 and 5 or something like that.